notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    data_dir = 'notebook/data/data_pre_ind_2/' # Location of the data (Should be changed to something more logical as here it implicates that only the preindustrial data is there)
</pre><p>Agree with in-code comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>

Reply:
closing conversation, resolved